conanfile.py

```diff
@@ -9,10 +9,11 @@ class LibsodiumConan(ConanFile):
     name = "libsodium"
     version = "1.0.15"
     url = "https://github.com/bincrafters/conan-libsodium"
+    homepage = "https://github.com/jedisct1/libsodium"
     description = "Sodium is a modern, easy-to-use software library for encryption, decryption, signatures, " \
                   "password hashing and more."
     license = "https://github.com/jedisct1/libsodium/blob/master/LICENSE"
-    exports_sources = ["LICENSE", "FindSodium.cmake"]
```

Comment:
this shouldn't be removed

Reply:
Some messy occurred during my rebase, thanks for review

Reply:
done!